verticapy/tests_new/machine_learning/vertica/test_base_model_methods.py

```diff
@@ -0,0 +1,1294 @@
+        ("Kurtosis", None, 1e-2, ABS_TOLERANCE),
+        ("Skewness", None, 1e-3, ABS_TOLERANCE),
+        ("Jarque-Bera (JB)", None, 1e-2, ABS_TOLERANCE),
+    ],
+)
+anova_report_args = (
+    "metric, metric_types, _rel_tolerance, _abs_tolerance",
+    [
+        ("df", ("dfr", "dfe"), rel_tolerance_map, ABS_TOLERANCE),
+        ("ss", ("ssr", "sse"), rel_tolerance_map, ABS_TOLERANCE),
+        ("ms", ("msr", "mse"), rel_tolerance_map, ABS_TOLERANCE),
+        ("f", ("f", ""), rel_tolerance_map, ABS_TOLERANCE),
+        ("p_value", ("p_value", ""), rel_tolerance_map, ABS_TOLERANCE),
+    ],
+)
+
+regression_metrics_args = (
+    "vpy_metric_name, py_metric_name, _rel_tolerance",
+    [
+        (("explained_variance", None), "explained_variance_score", rel_tolerance_map),
+        (("max_error", None), "max_error", rel_tolerance_map),
+        (("median_absolute_error", None), "median_absolute_error", rel_tolerance_map),
+        (("mean_absolute_error", None), "mean_absolute_error", rel_tolerance_map),
+        (("mean_squared_error", None), "mean_squared_error", rel_tolerance_map),
+        (("mean_squared_log_error", None), "mean_squared_log_error", rel_tolerance_map),
+        (
+            (
+                "rmse",
+                "root_mean_squared_error",
+            ),
+            "rmse",
+            rel_tolerance_map,
+        ),
+        (("r2", None), "r2_score", rel_tolerance_map),
+        (("r2_adj", None), "rsquared_adj", rel_tolerance_map),
+        (("aic", None), "aic", rel_tolerance_map),
+        (("bic", None), "bic", rel_tolerance_map),
+    ],
+)
+
+classification_metrics_args = (
+    "vpy_metric_name, py_metric_name, _rel_tolerance",
+    [
+        (("auc", None), "auc", rel_tolerance_map),
+        (("prc_auc", None), "prc_auc", rel_tolerance_map),
+        (("accuracy", None), "accuracy_score", rel_tolerance_map),
+        (
+            ("log_loss", None),
+            "log_loss",
+            rel_tolerance_map,
+        ),  # vertica uses log base 10, sklean uses natural log(e)
+        (("precision", None), "precision_score", rel_tolerance_map),
+        (("recall", None), "recall_score", rel_tolerance_map),
+        (("f1_score", None), "f1_score", rel_tolerance_map),
+        (("mcc", None), "matthews_corrcoef", rel_tolerance_map),
+        # (("informedness", None), "informedness", rel_tolerance_map), # getting mismatch for xgb
+        (("markedness", None), "markedness", rel_tolerance_map),
+        (("csi", None), "critical_success_index", rel_tolerance_map),
+    ],
+)
+
+
+@pytest.fixture
+def model_params(model_class):
+    """
+    fixture - model parameters
+    """
+    model_params_map = {
+        "RandomForestRegressor": (
+            "n_estimators, max_features, max_leaf_nodes, sample, max_depth, min_samples_leaf, min_info_gain, nbins",
+            [
+                # (None, None, None, None, None, None, None, None),  # accuracy does not mach with default parameters
```

Comment:
is this a copy mistake or actual error?
# (None, None, None, None, None, None, None, None),  # accuracy does not mach with default parameters

Reply:
None is for default value of parameter. I have commented it as verticapy accuracy is not matching with sklearn with default param. Also, verticapy and sklearn default param values are not same for few param. so , this test does not make sense.